perun-core/src/main/java/cz/metacentrum/perun/core/api/GroupsManager.java

```diff
@@ -832,6 +832,7 @@ public interface GroupsManager {
 	 * @param specificAttributes list of specified attributes which are needed in object richUser
 	 * @param allUserAttributes if true, get all possible user attributes and ignore list of specificAttributes (if false, get only specific attributes)
 	 * @param onlyDirectAdmins if true, get only direct user administrators (if false, get both direct and indirect)
+	 * @param role specified role
```

Comment:
How about changing this to a list/set of role names and then returning users with any of the roles (this would reduce calls in the GUI app, no?)

Reply:
I created it this way according to a planned usage in GUI - you can choose there just only one role at the same time when displaying managers. If we had a use case for getting all roles/more roles together, I would change it (but now I don't see this use case).

Reply:
I think the list would allow more flexibility, but also it can be updated later if necessary. Up to you